plugins/flytekit-envd/flytekitplugins/envd/image_builder.py

```diff
@@ -52,6 +52,8 @@ def create_envd_config(image_spec: ImageSpec) -> str:
     packages = [] if image_spec.packages is None else image_spec.packages
     apt_packages = [] if image_spec.apt_packages is None else image_spec.apt_packages
     env = {"PYTHONPATH": "/root", _F_IMG_ID: image_spec.image_name()}
+    commands = [] if image_spec.commands is None else image_spec.commands
```

Comment:
Commands run before or after packages? I'm assuming that apt packages run before commands.

Reply:
Command is always run after packages even if you change the order here. 
`apt_packages` -> `packages` -> `command`